common/src/main/java/com/sequenceiq/cloudbreak/auth/altus/Crns.java

```diff
@@ -0,0 +1,23 @@
+package com.sequenceiq.cloudbreak.auth.altus;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.sequenceiq.cloudbreak.auth.altus.Crn.ResourceType.MACHINE_USER;
+import static com.sequenceiq.cloudbreak.auth.altus.Crn.ResourceType.USER;
+import static com.sequenceiq.cloudbreak.auth.altus.Crn.Service.IAM;
+
+import java.util.Objects;
+
+public class Crns {
```

Comment:
are you sure we need a separate class for this one method? I think Crn class also can have it. Also the name of the class is too general especially when we see that the method is specialized for user related CRN

Reply:
I can put it into the Crn class.